Bold Subscriptions Customer Portal (#13)
bold-commerce/subscriptions-customer-portal: README.md

> # SUBSCRIPTIONS CUSTOMER PORTAL

Reviewer: ```suggestion
# Subscriptions Customer Portal
```

> 

Reviewer: Consider adding a description here that explains the purpose of this project.

Something like: 

```
## Description

This open source project serves as a starting point for developers creating a customer portal from scratch that is integrated with Bold Subscriptions.
```

> ## Local Set-Up

Reviewer: ```suggestion
## Installation

Use the following steps to set up and configure this project:
```

### Create a GitHub account if you do not have one already

> - https://github.com/

Reviewer: I think it can be assumed that someone has already done this and/or knows how. Let's delete this step.

> ### Fork and Clone the Repo

Reviewer: Rather than creating subheadings, let's just make this an ordered list.

> - Fork the repo into your workspace https://github.com/bold-commerce/subscriptions-customer-portal/tree/shopify

Reviewer: ```suggestion
1. Fork the repo into your workspace.
```

> - Clone your copy of the repo:
> - `$ git clone https://github.com/path/to/your/forked/version`

Reviewer: ```suggestion
2. Clone your copy of the repo using the following command:
    ```$ git clone https://github.com/path/to/your/forked/version```
```

> ### Ensure you are on the correct NODE version
> 

Reviewer: ```suggestion
3. Ensure you are on the correct Node version using the following commands:
```

> - `$ nvm install 16`
> - `$ nvm use 16`

Reviewer: ```suggestion
    ```
    $ nvm install 16
    $ nvm use 16
    ```
```

> ### Install, build, and run yarn
> 

Reviewer: ```suggestion
4. Install, build, and run yarn using the following commands:
```

> - `$ yarn`
> - `$ yarn build`
> - `$ yarn start`

Reviewer: ```suggestion
    ```
    $ yarn
    $ yarn build
    $ yarn start
    ```

> ### Navigate to window

Reviewer: ```suggestion
5. Navigate to the window with your local preview: `http://localhost:3000`
```

> - `http://localhost:3000`

Reviewer: ```suggestion
```

> ### How to setup on a Shopify theme

Reviewer: ```suggestion
## Configuration on a Shopify theme
```

> - On your shopify theme add this line to the bottom of your `theme.liquid` before the closing `</body></html>` tags

Reviewer: ```suggestion

1. In your Shopify theme, add the following line to the bottom of your `theme.liquid` before the closing `</body></html>` tags:
```

> ```html 
> <script src="http://localhost:3000/static/js/bundle.js" type="text/javascript"></script>
> ```

Reviewer: I think this needs to be indented to appear below the ordered line item:

```suggestion
     ```html 
    <script src="http://localhost:3000/static/js/bundle.js" type="text/javascript"></script>
    ```
```

> ### Insert div into subscription management page
> - Navigate to the page for managing subscriptions
> - Within the editor menu, click the insert HTML button and insert: 
>     
>     <div id="subscriptions-customer-portal-root"></div>

Reviewer: ```suggestion
2. Insert the following `<div>` into subscription management page
    1. Navigate to the page for managing subscriptions.
    2. Within the editor menu, click the **Insert HTML** button and insert: 
        ```
        <div id="subscriptions-customer-portal-root"></div>
        ```
```

> ### Setting up variables

Reviewer: I would recommend moving this step to the "Installation" list, preferably after the step telling the user to clone the repo.

> ### Setting up variables

Reviewer: Add a "Documentation" section at the end of the readme:

## Documentation

For more information, refer to the following resources:

- [Build a Customer Portal](https://developer.boldcommerce.com/default/guides/subscriptions-v2/customer-portals)
- [Bold Subscriptions API Specification](https://developer.boldcommerce.com/default/api/subscriptions)

> - Copy the .env.example -> .env, and update the value if required.

Reviewer: ```suggestion
Copy the `.env.example` file using the following command and update any values as necessary.
    ```
    cp .env.example .env
    ```
```
